autopsy_dashcam.py

```diff
@@ -52,29 +52,29 @@ def startUp(self, context):
         self.platform = ''
         pass
 
-    def process(self, file):
+    def process(self, filename):
```

Comment:
It's not strictly a ``filename`` (it has some more attributes), maybe ``source_file`` would be a nice variable. It's better than shadowing ``file`` in any case. Thanks for the contribution!

Reply:
Sorry about that! I can change quickly if you'd like, let me know @bartbroere

Reply:
Yes, if it's not too much effort. Thanks :sweat_smile:

Reply:
@bartbroere Updated!